igor.dzetcyna/first/task 1/hash.rb

```diff
@@ -0,0 +1,14 @@
+# Знайти кількість ключів, значення яких починається на “yes”
+hash = {
+  'Yes' => 23, 
+  'b' => 'travel', 
+  'yesterday' => 34, 
+  5 => '234', 
+  yesss: :fg, 
+  try: 30, 
+  key: 'some value', 
+  ' yesterday1 '=> 34,
+  ' yesteryear '=> 2014}
+number = hash.select { |keys| keys.to_s.downcase.start_with? "yes" }
+print "keys = "
```

Comment:
You could interpolate number.count to this string, but if there is no interpolation prefer single quoted strings.
https://github.com/rubocop-hq/ruby-style-guide#consistent-string-literals-single-quote